jvb/src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -561,40 +570,37 @@ public BridgeChannelMessage endpointMessage(EndpointMessage message)
             return null;
         }
 
-        boolean sendToOcto;
-
-        List<AbstractEndpoint> targets;
         if (message.isBroadcast())
         {
             // Broadcast message to all local endpoints + octo.
-            targets = new LinkedList<>(conference.getLocalEndpoints());
+            List<Endpoint> targets = new LinkedList<>(conference.getLocalEndpoints());
             targets.remove(endpoint);
-            sendToOcto = true;
+            conference.sendMessage(message, targets, /* sendToOcto */ true);
         }
         else
         {
             // 1:1 message
             String to = message.getTo();
 
             AbstractEndpoint targetEndpoint = conference.getEndpoint(to);
-            if (targetEndpoint instanceof OctoEndpoint)
+            if (targetEndpoint instanceof Endpoint)
+            {
+                ((Endpoint)targetEndpoint).sendMessage(message);
+            }
+            else if (targetEndpoint instanceof RelayedEndpoint)
             {
-                targets = Collections.emptyList();
-                sendToOcto = true;
+                ((RelayedEndpoint)targetEndpoint).getRelay().sendMessage(message);
```

Comment:
Why not expose `sendMessage` on `RelayedEndpoint` and the implementation of it just calls `getRelay` first?

Reply:
Because I don't want to accidentally iterate over all the `RelayedEndpoint`s and call `sendMessage` on each of them - forcing me to explicitly acknowledge I'm sending through the relay seems more likely to avoid that.